src/procedures/db.test.ts

```diff
@@ -56,11 +56,102 @@ describe("db procedures", () => {
                 RETURN this0"
             `);
             expect(params).toMatchInlineSnapshot(`
-{
-  "param0": "This is a lovely phrase",
-  "param1": "The Matrix",
-}
-`);
+                {
+                  "param0": "This is a lovely phrase",
+                  "param1": "The Matrix",
+                }
+            `);
+        });
+
+        test("Fulltext with options", () => {
+            const fulltextProcedure = Cypher.db.index.fulltext.queryNodes(
+                "my-text-index",
+                new Param("This is a lovely phrase"),
+                {
+                    skip: 5,
+                    analyser: new Param("whitespace"),
+                }
+            );
+
+            const { cypher, params } = fulltextProcedure.build();
+
+            expect(cypher).toMatchInlineSnapshot(
+                `"CALL db.index.fulltext.queryNodes(\\"my-text-index\\", $param0, { skip: 5, analyser: $param1 })"`
+            );
+            expect(params).toMatchInlineSnapshot(`
+                {
+                  "param0": "This is a lovely phrase",
+                  "param1": "whitespace",
+                }
+            `);
+        });
+    });
+    describe("db.index.fulltext.queryRelationships", () => {
+        test("Simple fulltext", () => {
+            const targetNode = new Cypher.Node({ labels: ["Movie"] });
```

Comment:
I guess this is misleading as I expect that the target should be of type Relationship in this case